monitoring/perf_flag.cc

```diff
@@ -0,0 +1,29 @@
+#include "rocksdb/perf_flag.h"
+
+namespace rocksdb {
+#ifdef ROCKSDB_SUPPORT_THREAD_LOCAL
+__thread uint8_t perf_flags[FLAGS_LEN];
+#else
+uint8_t perf_flags[FLAGS_LEN];
+#endif
+
+void EnablePerfFlag(uint64_t flag) {
+  if (CheckPerfFlag(flag)) {
+  } else {
+    GET_FLAG(flag) ^= (uint64_t)0b1 << ((uint64_t)flag & (uint64_t)0b111);
```

Comment:
Why do we need this part? `& (uint64_t)0b111`

Reply:
& 0b111 means find where is the flag located
which is equivalent to % 8 and faster